ROADMAP.md

```diff
@@ -41,6 +42,10 @@ Objective: "Improve the Serverless Experience by reducing cold starts/stops to 1
 * Server Shutdown
     * Ensure that all model servers shutdown within 10 seconds of not receiving traffic.
 
+Objective: "Simplify User Experience."
+* Secure storage mechanisms
+    * Explore simplifying user experience with storage backends protected by credentials (e.g. S3/GCS accounts with credentials)
```

Comment:
Is there something more actionable than "explore". Are we designing something? What specifically are we aiming to improve. Perhaps just "Address the storage pains discussed in #XYZ"

Reply:
Changed it to implement, and can be discussed how.